tests/tiglFuselageSegment.cpp

```diff
@@ -712,3 +712,55 @@ TEST_F(TiglFuselageSegmentSimple, getSectionCenter)
     ASSERT_EQ(TIGL_MATH_ERROR, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta_out_of_range, &pointX, &pointY, &pointZ));
 }
 
+TEST_F(TiglFuselageSegmentSimple, getSectionArea)
+{
+    // test first fuselage segment
+    double eta = 0.;
+
+    double area = 0.;
+
+    double pi = 3.14159265359;
+    ASSERT_NE(TIGL_NULL_POINTER, tiglFuselageGetCrossSectionArea(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &area));
+    ASSERT_EQ(TIGL_SUCCESS, tiglFuselageGetCrossSectionArea(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &area));
+    EXPECT_NEAR(area, pi * 0.5 * 0.5, 3e-2);
```

Comment:
EXPECT_NEAR works as follows:
EXPECT_NEAR(expected_value, actual_value, tolerance).

You switched the first two arguments. It better is: EXPECT_NEAR(M_PI*0.5*0.5, area, 3e-2)